themes/V3/5ePHB/style.less

```diff
@@ -786,6 +786,32 @@
 // *****************************
 // *      TABLE OF CONTENTS
 // *****************************/
+
+// Default Exclusions
+// Anything not exlcuded is included, default Headers are H1, H2, and H3.
+h4,
+h5,
+h6,
+.page:has(.frontCover), 
+.page:has(.backCover), 
+.page:has(.insideCover), 
+.monster,
+.noToC,
+.toc { --TOC: exclude; }
+
```

Comment:
Since `partCover` should *only* include the H1 header in the ToC, perhaps we can add a rule like this:

```
.page:has(.partCover) {
  --TOC: exclude
  &h1 {
    --TOC: include
  }
}
```